src/meteorite.ts

```diff
@@ -5,22 +5,30 @@ class Meteorite extends GameObject {
     constructor() {
         super();
         this.size = 0;
-        this.position = createVector(0, 0);
-        this.velocity = createVector(0, 0);
+        this.position = createVector(random(width), random(height));
+        this.velocity = createVector(10, 0);
     // this.acceleration = createVector(0, 0);
         this.damage = 0;
     // this.collisionSound = p5.SoundFile;        
     }
 
     update() {
-        
+        this.position.sub(this.velocity)
+        if (this.position.x < 0) {
+            this.position.x = width;
+            this.position.y = random(height);
+        }
     }
 
     draw() {
-
+        noStroke();
+        fill(color("red"));
+        ellipse(this.position.x, this.position.y, 10, 10);
+        push();
```

Comment:
push( ) ska ligga innan noStroke och pop( ) efter. Man kaplar in sina inställningar för att rita ut objektet med dessa funktioner